oarepo_communities/ui/communities_components/templates/oarepo/invenio_communities/search.html

```diff
@@ -0,0 +1,4 @@
+    {% extends "semantic-ui/invenio_communities/search.html" %}
```

Comment:
Is this template necessary?

Reply:
@mirekys Sorry my bad. Previously I had overrided a new community button there, but now the template is indeed useless.